corio.py

```diff
@@ -303,7 +310,7 @@ def main(options):
             activate_degraded_mode(options)
             options.number_of_nodes -= literal_eval(os.getenv("DEGRADED_PODS_CNT"))
         start_processes(processes)
-        while True:
+        while processes:
```

Comment:
This will be executed until all process will be completed so this change is as good as previous condition and more specific.